app/Component/Field/DateField.js

```diff
@@ -25,13 +25,23 @@ class InputField extends React.Component {
         if (value) {
             value = typeof(value) === 'string' ? moment(value, format) : moment(value);
         }
-        let onChange = this.onChange.bind(this);
 
-        return <DatePicker
-            selected={value}
-            dateFormat={format}
-            onChange={onChange}
-            />;
+        let attributes = {
+            dateTime: value,
+            format: format,
+            inputFormat: format,
+            onChange: this.onChange.bind(this)
+        };
+
+        if ('date' === this.props.field.type()) {
```

Comment:
According to the documentation the `mode` is undefined by default.
Is it working without the `mode` set to `time` for a `datetime` field ?